test/mainnet/configurer.spec.sol

```diff
@@ -149,4 +158,129 @@ contract VaultConfigureUpgradeTest is Test, MainnetActors {
+        vault.processAccounting();
+        // Assert totalAssets is correct
+        uint256 totalAssets = vault.totalAssets();
+
+        uint256 ynLSDeRate = IProvider(vault.provider()).getRate(MC.YNLSDE);
+        assertEqThreshold(totalAssets, totalAssetBefore + (depositAmount * ynLSDeRate / 1e18), 5,"Total assets should match deposit amount");
+    }
+
+    function test_deposit_wETH() public {
+     uint256 totalAssetBefore = vault.totalAssets();
+        uint256 depositAmount = 1 ether;
+
+        deal(MC.WETH,address(this), depositAmount);
+        // Deposit YN
+          IERC20(MC.WETH).approve(address(vault), depositAmount);
+        vault.depositAsset(MC.WETH, depositAmount, address(this));
+        vault.processAccounting();
+        // Assert totalAssets is correct
+        uint256 totalAssets = vault.totalAssets();
+
+        uint256 WETHRate = IProvider(vault.provider()).getRate(MC.WETH);
+        assertEqThreshold(totalAssets, totalAssetBefore + (depositAmount * WETHRate / 1e18), 5,"Total assets should match deposit amount");
+    }
+
+    function test_donate_whitlisted_Assets() public {
+        uint256 donationAmount = 1 ether;
+        uint256 totalAssetBefore = vault.totalAssets();
+        
+        // Donate the 10 whitelisted assets to the vault
+        address[] memory whitelistedAssets = new address[](10);
+        whitelistedAssets[0] = MC.WETH;
+        vm.label(MC.WETH, "WETH");
+        deal(MC.WETH, address(this), donationAmount * 10);
+        whitelistedAssets[1] = MC.YNETH;
+        vm.label(MC.YNETH, "YNETH");
+        deal(MC.YNETH, address(this), donationAmount);
+
+        whitelistedAssets[2] = MC.YNLSDE;
+        vm.label(MC.YNLSDE, "YNLSDE");
+        deal(MC.YNLSDE, address(this), donationAmount);
+
+        whitelistedAssets[3] = MC.STETH;
+        {
+            // Deal  to and convert to stETH
+            vm.deal(address(this), donationAmount);
+            (bool success,) = MC.STETH.call{value: donationAmount}("");
+            assertTrue(success, "stETH deposit failed");
+            vm.stopPrank();
+        }
+        vm.label(MC.STETH, "STETH");
+
+        whitelistedAssets[4] = MC.EULER_WETH_22_VAULT;
+        vm.label(MC.EULER_WETH_22_VAULT, "EULER_WETH_22_VAULT");
+        deal(MC.EULER_WETH_22_VAULT, address(this), donationAmount);
+
+        whitelistedAssets[5] = MC.CURVE_LP_YNETH_YNLSDE_STRATEGY;
+        vm.label(MC.CURVE_LP_YNETH_YNLSDE_STRATEGY, "CURVE_LP_YNETH_YNLSDE_STRATEGY");
+        deal(MC.CURVE_LP_YNETH_YNLSDE_STRATEGY, address(this), donationAmount);
+
+        whitelistedAssets[6] = MC.WSTETH;
+        vm.label(MC.WSTETH, "WSTETH");
+        deal(MC.WSTETH, address(this), donationAmount);
+        
+        whitelistedAssets[7] = MC.OETH;
+        vm.label(MC.OETH, "OETH");
+        // {
+        //  //TODO: figure out how to get OETH this reverts
+        //     IERC20(MC.WETH).approve(MC.OETH, donationAmount);
```

Comment:
needs to fix this

Reply:
fixed in [900f7b4](https://github.com/yieldnest/yieldnest-vault/pull/101/commits/900f7b459f7c0d067afd5ee0e90f2fc1ab85cd78)